example/src/main/java/com/dsc/formbuilder/screens/survey/components/PersonalDetails.kt

```diff
@@ -78,9 +107,13 @@ fun TextInput(label: String, state: TextFieldState) {
 fun PersonalDetailsPreview() {
     val formState: FormState<BaseState<*>> = FormState(
         listOf(
-            TextFieldState("username"),
             TextFieldState("email"),
-            TextFieldState("number")
+            TextFieldState("phone"),
+            TextFieldState(
+                "date",
+                formatter = DateFormatter(dateFormat = DateFormat.DDMMYYYY, separator = "/"),
```

Comment:
Awesome to see that the preview is actually interactive and applies our formatters 😎